.github/workflows/pr-build.yml

```diff
@@ -37,9 +64,54 @@ jobs:
       - name: Build with Flutter
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        run: flutter build apk --debug
+        run: |
+          flutter build apk --${{ inputs.app-variant }};
+      - name: Set env
+        run: |
+          echo "COMMIT_HASH=$(git rev-parse --short HEAD)" >> $GITHUB_ENV
       - name: Upload build
         uses: actions/upload-artifact@v3
         with:
-          name: revanced-manager
+          name: revanced-manager-${{ env.COMMIT_HASH }}-P${{ inputs.pr-number }}-B${{ inputs.app-variant }}-F${{ inputs.flutter-variant }}
```

Comment:
What is the purpose of P, B and F?

Reply:
Just a placeholder, will delete soon

- P: Pull Request
- B: Build variant
- F: Flutter variant